business_requirement/models/business.py

```diff
@@ -40,7 +40,7 @@ def _get_default_company(self):
         states={'draft': [('readonly', False)]}
     )
     ref = fields.Char(
-        'Reference',
+        'WBS',
         required=False,
         readonly=True,
```

Comment:
Hmm.. I haven't changed it was already there.

But yes, I think read-only should not be there, should I remove?